.eslintrc.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+    // Enforcing ES6 and Node environments
+    env: {
+        es6: true,
+        node: true
+    },
+    // Using the airbnb base configuration
+    extends: [
+        'eslint-config-airbnb-base/rules/best-practices',
+        'eslint-config-airbnb-base/rules/errors',
+        'eslint-config-airbnb-base/rules/node',
+        'eslint-config-airbnb-base/rules/style',
+        'eslint-config-airbnb-base/rules/variables',
+        'eslint-config-airbnb-base/rules/es6',
+        'eslint-config-airbnb-base/rules/strict',
+    ].map(require.resolve),
+    // Define the global variables we use
+    globals: {
+        assert: true,
+        browser: true,
+        expect: true,
+    },
```

Comment:
If we add `mocha` or `jasmine` as environment we don't need to set `assert` and `expect` as globals